YoutubeExplode/YoutubeClient.cs

```diff
@@ -40,8 +40,17 @@ public class YoutubeClient
     /// </summary>
     public YoutubeClient(HttpClient http, IReadOnlyList<Cookie> initialCookies)
     {
-        var youtubeHttp = new HttpClient(new YoutubeHttpHandler(http, initialCookies), true);
+        // Add necessary cookies.
+        var cookies = new List<Cookie>(initialCookies)
+        {
+            new("SOCS", "CAESEwgDEgk0ODE3Nzk3MjQaAmVuIAEaBgiA_LyaBg") { Domain = "youtube.com" } //cookie consent request bypass
+        };
 
+        var youtubeHttp = new HttpClient(
+            new YoutubeHttpHandler(http, cookies),
+            true
+        );
```

Comment:
Please move this logic here instead:

https://github.com/Tyrrrz/YoutubeExplode/blob/f4ff634f9dfebd5446f5a27a7f318b0adbf7bdff/YoutubeExplode/YoutubeHttpHandler.cs#L27-L29

Add the cookie in `YoutubeHttpHandler`'s constructor